js/test/backstop/backstop_data/engine_scripts/puppet/onReady.js

```diff
@@ -1,4 +1,5 @@
 module.exports = async (page, scenario, vp) => {
   console.log('SCENARIO > ' + scenario.label);
+  if(scenario.enableRebrand) await require('./enableRebrand')(page)
```

Comment:
doesn't update the svgs in the header/footer or the favicon but that's not important for our tests